docs/usage/self-hosted-configuration.md

```diff
@@ -830,7 +830,7 @@ Secret names must start with an upper or lower case character and can have only
 
 By default, Renovate will use the most efficient approach to updating package files and lock files, which in most cases skips the need to perform a full module install by the bot.
 If this is set to false, then a full install of modules will be done.
-This is currently applicable to `npm` and `lerna`/`npm` only, and only used in cases where bugs in `npm` result in incorrect lock files being updated.
+This is currently applicable to `npm` only, and only used in cases where bugs in `npm` result in incorrect lock files being updated.
```

Comment:
Are you sure the PR title is correct? I think we're missing an `!` somewhere, to denote the `major` release that the Semantic Release bot should create after this PR gets merged into `main`?

Reply:
So far this v37 branch contains only one commit/change, but it will be expanded on